.github/workflows/license_check.yml

```diff
@@ -18,4 +18,4 @@ jobs:
     steps:
     - uses: actions/checkout@v3
     - name: Check License Header
-      uses: apache/skywalking-eyes/header@df70871af1a8109c9a5b1dc824faaf65246c5236
+      uses: apache/skywalking-eyes@775fe1ffda59b7e100aa144d0ef8d7beae17f97d
```

Comment:
`5b7ee1731d036b5aac68f8bd3fc9e6f98ada082e` introduced cache, but I tested many times and cannot understant why it doesn't work. Anyway this can work. 😅